sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/DistributeTest.java

```diff
@@ -448,6 +448,21 @@ public void doNotShowUpdateSetupFailedDialogInBackground() {
         SharedPreferencesManager.remove(PREFERENCE_KEY_UPDATE_SETUP_FAILED_MESSAGE_KEY);
     }
 
+    @Test
+    public void doNotShowUpdateSetupFailedDialogInNullForegroundActivity() {
+        when(SharedPreferencesManager.getString(PREFERENCE_KEY_UPDATE_SETUP_FAILED_MESSAGE_KEY)).thenReturn("failed_message");
```

Comment:
I think we don't need to set this preference, as we only call `showUpdateSetupFailedDialog`